operations/rates.md

```diff
@@ -169,7 +251,7 @@ The price in the response is dependent on the enterprise's [pricing](configurati
 | :-- | :-- | :-- | :-- |
 | `Currency` | string | required | ISO-4217 code of the [Currency](currencies.md#currency). |
 | `TimeUnitStartsUtc` | array of string | required | Set of all time units covered by the time interval; expressed in UTC timezone ISO 8601 format. |
-| `BasePrices` | array of number | required | Base prices of the rates for each time unit covered by the time interval. |
+| `BaseAmountPrices` | [Amount value](accountingitems.md#amount-value) | required | Base prices of the rates for each time unit covered by the time interval. |
```

Comment:
I think this should be "array of [Amount value](accountingitems.md#amount-value)"